R/methods-xcmsRaw.R

```diff
@@ -664,6 +672,419 @@ setMethod("findPeaks.centWave", "xcmsRaw", function(object, ppm=25, peakwidth=c(
 })
 
 ############################################################
+## findPeaks.centWaveWithPredictedIsotopeROIs
+setMethod("findPeaks.centWaveWithPredictedIsotopeROIs", "xcmsRaw", function(object, ppm=25, peakwidth=c(20,50), snthresh=10,
```

Comment:
I know you're just following the other examples in `xcms` but I would opt against passing a list of arguments like you do in `addROIsParams`. It's really hard to check these for proper input and it is also not always clear to the user how to pass arguments correctly. What about passing them as individual parameters?